plugins/chat/app/controllers/chat/api/channels_messages_interactions_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Chat::Api::ChannelsMessagesInteractionsController < Chat::ApiController
+  def create
+    Chat::CreateMessageInteraction.call(service_params) do |result|
+      on_success do
+        render_serialized(
+          result.interaction,
```

Comment:
Not saying that’s a better way of doing it, but as an alternative, you could do:
```ruby
on_success do |interaction:|
  render_serialized(interaction, …)
end
```
(and drop the `|result|` from line 5). Up to you :slightly_smiling_face:

Reply:
ah yes I forgot about this, it's fairly recent right?